src/settings.ts

```diff
@@ -12,8 +12,8 @@ export class MySettings {
     }
 
     private static getConfigurationValue(key: string) {
-        let configuration = vscode.workspace.getConfiguration("elrond");
+        let configuration = vscode.workspace.getConfiguration("multiversx");
```

Comment:
So existing configurations won't be migrated.

Perhaps add a fallback or something like that? (not sure about impact, though)